targets/TARGET_Samsung/TARGET_SIDK_S5JS100/TOOLCHAIN_GCC_ARM/.gitattributes

```diff
@@ -0,0 +1,4 @@
+*.o binary
+*.lib binary
+*.a binary
```

Comment:
I'll leave the comment here for libSAMSUNG_ libs. They need license files , following https://os.mbed.com/docs/mbed-os/latest/contributing/license.html